ports/freebsd/Makefile

```diff
@@ -0,0 +1,49 @@
+# Created by: Donal Cahill <din@ury.york.ac.uk>
+# $FreeBSD$
+
+PORTNAME=		ury-playd
+DISTVERSIONPREFIX=	v
+DISTVERSION=		0.3.1
+CATEGORIES=		audio
+
+MAINTAINER=	din@ury.york.ac.uk
```

Comment:
I use `@ury.york.ac.uk` on GitHub, so I have to use it for my commits at least. I'll change it here. I'll have to look changing my GitHub too.